src/utilities/make_state_vector_file.py

```diff
@@ -181,26 +190,321 @@ def make_state_vector_file(
     # Save
     if save_pth is not None:
         print("Saving file {}".format(save_pth))
-        ds_statevector.to_netcdf(
-            save_pth,
-            encoding={
-                v: {"zlib": True, "complevel": 9} for v in ds_statevector.data_vars
-            },
-        )
+        ds_statevector.to_netcdf(save_pth)
 
     return ds_statevector
 
 
+
+def get_country_code(ROI):
+    """
+    Matches the given country to its country code by searching through online database
+
+    Arguments
+        ROI            [str]  : Name of the country specified in config file
+
+    Returns
+        countryCode    [str]  : Three-letter country code (e.g. USA) of the ROI
+    """
+    geodata_url = "https://gadm.org/download_country.html"  # URL of the database for country shapefiles
+
+    try:
+        # Send a GET request to the geodata URL with a timeout of 5 seconds
+        page = requests.get(geodata_url, timeout=5)
+
+        # Create a BeautifulSoup parser object to parse the HTML content
+        parser = BeautifulSoup(page.content, 'html.parser')
+
+        found = False  # Flag to track if the ROI is found in the database
+
+        # Iterate over each 'option' tag in the parsed HTML content
+        for country in parser.find_all('option'):
+            if country.string == ROI:  # Check if the country name matches the given ROI
+                countryInfo = str(country)
+                # Extract the country code from the HTML string
+                countryCode = countryInfo.split("=\"")[1].split("_")[0]
+                print(countryCode)  # Print the country code
```

Comment:
Either remove the print statement or make it more informative. `print(f"{ROI} country found in database with country code: {countryCode}")`